dataeng/resources/warehouse-transforms-ci.sh

```diff
@@ -43,6 +43,9 @@ pip install -r requirements.txt
 if [ "$isReporting" == "true" ]
 then
 
+    # Overide the commit author in case of reporting
+    NOTIFY=${ghprbActualCommitAuthorEmail}
```

Comment:
I am trying to overwrite NOTIFY email address to Commit author email authors, commit author emails do appear but NOTIFY variable is not overwritten, Is is it possible if we have some variable defined in DSL and can be overwritten inside shell script?

Reply:
@bmedx